### PR TITLE
type of impostor in login message

### PR DESCRIPTION
**Pull request by ajaxvp**
this doesnt need to be approved but i just saw it and thought it was a neat idea i mean idk

https://totalfreedom.boards.net/thread/63314/mistake-verifying-op-again-3?page=1&scrollTo=723610

### CODE REVIEW

**Comment by ncurran02** [comment]
There’s much neater way to do this.

**Comment by ajaxvp** [comment]
> There’s much neater way to do this.

i know, i wasn't really going for neatness, though

**Comment by ajaxvp** [comment]
we're disabling it entirely instead.

File: TotalFreedomMod.iml [hunk 1]
```diff
@@ -12,17 +12,22 @@
     <orderEntry type="sourceFolder" forTests="false" />
```
Comment (lemonsked) [inline TotalFreedomMod.iml]: can we not edit this file

Comment (ajaxvp) [reply TotalFreedomMod.iml]: yes

**Pull request merged**